Sources/Benchmark/BenchmarkReporter.swift

```diff
@@ -18,9 +18,14 @@ protocol BenchmarkReporter {
     mutating func report(running name: String, suite: String)
     mutating func report(finishedRunning name: String, suite: String, nanosTaken: UInt64)
     mutating func report(results: [BenchmarkResult])
+    
+    var columns: [Column] { get set }
```

Comment:
Are you sure this is necessary here? I think you might be able to simplify a bit if this isn't required.